src/main/osd/osd.h

```diff
@@ -78,7 +78,7 @@ extern const char * const osdTimerSourceNames[OSD_NUM_TIMER_TYPES];
 #define OSD_TYPE_MASK           0xC000  // bits 14-15
 #define OSD_POS(x,y)  ((x & OSD_POSITION_XY_MASK) | ((x << (OSD_POSITION_BIT_XHD - OSD_POSITION_BITS)) & OSD_POSITIION_XHD_MASK) | \
                        ((y & OSD_POSITION_XY_MASK) << OSD_POSITION_BITS))
-#define OSD_X(x)      ((x & OSD_POSITION_XY_MASK) | ((x & OSD_POSITION_BIT_XHD) >> (OSD_POSITION_BIT_XHD - OSD_POSITION_BITS)))
+#define OSD_X(x)      ((x & OSD_POSITION_XY_MASK) | ((x & (1 << OSD_POSITION_BIT_XHD)) >> (OSD_POSITION_BIT_XHD - OSD_POSITION_BITS)))
```

Comment:
I think we should use `OSD_POSITIION_XHD_MASK` here

Reply:
I agree, and I'll also fix the spelling.

Reply:
well spotted! I didn't see that lol 😄

Reply:
Missed that one too. Need a bigger screen for review :)